types/koa-csrf/index.d.ts

```diff
@@ -1,41 +1,40 @@
-// Type definitions for koa-csrf 3.0
+// Type definitions for koa-csrf 5.0
 // Project: https://github.com/koajs/csrf
 // Definitions by: Haskaalo <https://github.com/haskaalo>
+//                 Kirill Shvets <https://github.com/kirillshvets97>
```

Comment:
Are you looking to become a package owner/reviewer, or are just trying to indicate contribution credit? If the latter, you don't need to modify this section: https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
I actively use Koa at work, so I can become a package owner/reviewer, why not